### PR TITLE
Format using nph

### PR DESCRIPTION
**Pull request by bhartnett**
This is just a formatting change using `nph .` and a minor change to add the nph lint check to CI.

### CODE REVIEW

**Comment by bhartnett** [comment]
@arnetheduck Just a heads up that the latest tagged/released version of nph doesn't work with the latest version of nim. The fix is in the master branch but not released yet. This issue should probably be prioritized: https://github.com/arnetheduck/nph/issues/70

**Comment by arnetheduck** [comment]
> Just a heads up that the latest tagged/released version of nph doesn't work with the latest version of nim.

yeah, I know - as long as it's built on its own via nimble (which is the supported way of building it), it should be fine though.

**Review by arnetheduck — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,6 +7,27 @@ on:
...
+
+      - name: Check nph formatting
+        # Pin nph to a specific version to avoid sudden style differences.
+        # Updating nph version should be accompanied with running the new
+        # version on the project directory.
+        run: |
+          VERSION="v0.5.1"
+          ARCHIVE="nph-linux_x64.tar.gz"
+          curl -L "https://github.com/arnetheduck/nph/releases/download/${VERSION}/${ARCHIVE}" -o ${ARCHIVE}
+          tar -xzf ${ARCHIVE}
```
Comment (kdeme) [inline .github/workflows/ci.yml:27]: Perhaps it should instead become a dependency in the nimble file? Did you try if that works?

**Comment by bhartnett** [comment]
> > Just a heads up that the latest tagged/released version of nph doesn't work with the latest version of nim.
> 
> yeah, I know - as long as it's built on its own via nimble (which is the supported way of building it), it should be fine though.

Yes I see what you mean. Using nim version 2.0.6 when I run `nimble install nph`, it fails to install the latest tagged version so it falls back to installing the master version which does work.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bhartnett) [reply .github/workflows/ci.yml:27]: Sure, yes it works locally for me. I'll try that

**Pull request merged**